AirCasting/Map/GoogleMapView.swift

```diff
@@ -245,10 +245,19 @@ struct GoogleMapView: UIViewRepresentable {
         }
         
         func centerMap(for mapView: GMSMapView) {
-            let camera = GMSCameraPosition.camera(withLatitude: parent.tracker.locationManager.location!.coordinate.latitude,
-                                                  longitude: parent.tracker.locationManager.location!.coordinate.longitude,
-                                                  zoom: 16)
-            mapView.animate(to: camera)
+            switch parent.liveModeOn {
```

Comment:
Switch looks a bit weird to me here, have you considered something like:
```Swift
let lat = parent.liveModeOn ? 
     parent.tracker.locationManager.location!.coordinate.latitude : 
     parent.pathPoints.last?.location.latitude ?? 37.35
let long = parent.liveModeOn ? 
     parent.tracker.locationManager.location!.coordinate.longitude : 
     parent.pathPoints.last?.location.longitude ?? -122.05
let camera = GMSCameraPosition(withLatitude: lat, longitude: long, zoom: 16)
mapView.animate(to: camera)
```

Reply:
Really I need to say no. Sometimes I try to be cool using cool stuff - the reason why switch 😅. I will change it to yours, Thank you!